docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "3"
+services:
+  rails: &rails_base
+    build:
+      dockerfile: infra/dev/rails/Dockerfile
+      context: .
+    ports:
+      - "8080:8080"
+    depends_on:
+      - db
+    environment:
+      ALGOLIASEARCH_API_KEY: ${ALGOLIASEARCH_API_KEY}
+      ALGOLIASEARCH_APPLICATION_ID: ${ALGOLIASEARCH_APPLICATION_ID}
+      ALGOLIASEARCH_SEARCH_ONLY_KEY: ${ALGOLIASEARCH_SEARCH_ONLY_KEY}
+      GITHUB_KEY: ${GITHUB_KEY}
+      GITHUB_SECRET: ${GITHUB_SECRET}
+      RAILS_ENV: development
+      DATABASE_URL: postgres://postgres:mysecretpassword@db:5432/postgres
+    volumes:
+      - .:/usr/src/app
+    command: bin/rails server -b 0.0.0.0 -p 8080
+  redis:
+    image: redis:3.2.11
+    ports:
+      - "6379:6379"
+  migrate:
+    ports: []
+    <<: *rails_base
+    command: bin/rails db:migrate
```

Comment:
Should there be a step for db:setup for the first run? Might be a documentation fix